src/rtapi/uspace_rtapi_app.cc

```diff
@@ -919,6 +922,7 @@ int Posix::task_delete(int id)
 static int find_rt_cpu_number() {
     if(getenv("RTAPI_CPU_NUMBER")) return atoi(getenv("RTAPI_CPU_NUMBER"));
 
+#ifdef __linux__
     cpu_set_t cpuset_orig;
     int r = sched_getaffinity(getpid(), sizeof(cpuset_orig), &cpuset_orig);
```

Comment:
hm a little googling suggests that freebsd has pthread_setaffinity_np: https://www.freebsd.org/cgi/man.cgi?manpath=FreeBSD%208.2-RELEASE&query=pthread_setaffinity_np 
as does Linux.  With the rtapi_cpu_set_t typedef, can this function be reworked to not have any `#ifdef`s?